### ISSUE TITLE
training script is stuck in bmt.init_distributed(seed=0) 

### ISSUE THREAD

**Issue by greenteaofwhu** [OpenBMB/BMTrain#72]
GPU：V100
torch version：1.10.0+cu113
Python 3.7.13
bmtrain：0.2.1
![image](https://user-images.githubusercontent.com/29090584/217821166-a8692c8d-8ebf-4324-8189-1809fc4692fb.png)
![image](https://user-images.githubusercontent.com/29090584/217821807-8ad15780-1a6f-467e-bf38-cc35c444c376.png)

**Comment by a710128** [comment]
Have you tried putting init_distributed in front of get_args?

**Comment by wyl7** [comment]
also have the same question

**Comment by wyl7** [comment]
> GPU：V100 torch version：1.10.0+cu113 Python 3.7.13 bmtrain：0.2.1 ![image](https://user-images.githubusercontent.com/29090584/217821166-a8692c8d-8ebf-4324-8189-1809fc4692fb.png) ![image](https://user-images.githubusercontent.com/29090584/217821807-8ad15780-1a6f-467e-bf38-cc35c444c376.png)

Hello greenteaofwhu😁, do you solve this issue?

**Comment by MayDomine** [comment]
Please follow the instruction of bug issue template, and provide all details needed for reproduce your error.
